test/Microsoft.AspNet.Razor.Test/Parser/Html/HtmlAttributeTest.cs

```diff
@@ -329,12 +362,12 @@ public void ConditionalAttributesDoNotCreateExtraDataForEntirelyLiteralAttribute
         [Fact]
         public void ConditionalAttributesAreDisabledForDataAttributesInBlock()
         {
-            ParseBlockTest("<span data-foo='@foo'></span>",
+            ParseBlockTest("<span data-foo  =  '@foo'></span>",
```

Comment:
Why not do both the old and new cases? Same for `ConditionalAttributesAreDisabledForDataAttributesInDocument()`.